scripts/simulate_agents.py

```diff
@@ -0,0 +1,150 @@
+
+        for module in active_modules:
+            if module not in available_modules:
+                raise ValueError(f"Selected module: '{module}' doesn't exist on agent simulator!")
+
+        logger.info(agent.modules)
+
+        active_agents.append(agent)
+        sender = ag.Sender(manager_address, protocol=protocol)
+        injectors.append(ag.Injector(sender, agent))
+
+    try:
+        start(injectors)
+        sleep(run_duration)
+    finally:
+        stop(injectors)
+
+
+def start(agent_injectors):
+    print(f"Running {len(agent_injectors)} injectors...")
+    for injector in agent_injectors:
+        injector.run()
+
+
+def stop(agent_injectors):
+    print(f"Stopping {len(agent_injectors)} injectors...")
+    for injector in agent_injectors:
+        injector.stop_receive()
+
+
+if __name__ == "__main__":
+    arg_parser = argparse.ArgumentParser()
+
+    arg_parser.add_argument('-a', '--manager', metavar='<manager_ip_address>', type=str, required=False,
+                            default='localhost', help='Manager IP address', dest='manager_addr')
+
+    arg_parser.add_argument('-e', '--eps', metavar='<events_per_second>', type=int, required=False, default=1000,
+                            help='Number of events per second to be generated by each agent', dest='eps')
+
+    arg_parser.add_argument('-n', '--agents', metavar='<agents_number>', type=int, default=5, required=False,
+                            help='Number of agents to create and run', dest='n_agents')
+
+    arg_parser.add_argument('-b', '--batch', metavar='<agents_batch>', dest='b_agents',
+                            type=int, required=False, default=2, help='Number of agents to create on each process')
+
+    arg_parser.add_argument('-o', '--os', metavar='<os>', dest='os',
+                            type=str, required=False, default='debian8', help='Agent operating system',)
+
+    arg_parser.add_argument('-p', '--protocol', metavar='<protocol>', dest='agent_protocol',
+                            type=str, required=False, default=TCP, help='Communication protocol')
+
+    arg_parser.add_argument('-t', '--time', metavar='<monitoring_time>', dest='duration',
+                            type=int, required=False, default=20, help='Time in seconds for monitoring')
+
+    arg_parser.add_argument('-v', '--version', metavar='<version>', dest='version',
+                            type=str, required=False, default='4.2.0', help='Agent wazuh version', )
+
+    arg_parser.add_argument('-m', '--modules', dest='modules', required=False, type=str, nargs='+', action='store',
+                            default=['fim'], help='Active module separated by whitespace.')
+
+    arg_parser.add_argument('-s', '--modules-eps', dest='modules_eps', required=False, type=int, nargs='+',
+                            action='store', default=None, help='Active module EPS separated by whitespace.')
+
+    args = arg_parser.parse_args()
+
+    if args.b_agents > 2:
```

Comment:
Use a better name for this variable. Someone who isn't familiar with the code/tool may find this hard to read.

Reply:
https://github.com/wazuh/wazuh-qa/pull/1148/commits/2f8cf6f1bc8896a82530968320a50c3d7c7edd84